packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -327,11 +326,12 @@ pub trait EntityStore: crud::Read<Entity> {
     /// - if an [`EntityUuid`] was supplied and already exists in the store
     async fn create_entity(
         &mut self,
-        properties: EntityProperties,
+        entity_id: EntityId,
```

Comment:
Why have you changed this to be required? 🤔

Reply:
This was to align with the Postgres implementation. I expect to only return newly generated parameters from this function in the future and as the entity id can be known beforehand it feels more intuitive to pass it directly. Also, this encapsulates better, that the `owned_by_id` and `entity_uuid` belongs together. Starting with temporal versioning, we also store them next to each other in the database.

Reply:
I don't like this change being a drive-by in this PR if I'm honest. Can we revert it, and open a separate discussion in Slack to discuss this. It's quite a foundational change to say that the calling applications need to be responsible for generating EntityId's. It's something we've discussed before, but not agreed on, or decided the ramifications of.

Reply:
Reverted in 7f7fa9e0992b21605cfc296da69c41af8c76c839